src/SIL.XForge.Scripture/ClientApp/src/app/app.component.html

```diff
@@ -132,6 +132,12 @@
               {{ t("serval_administration") }}
             </button>
           }
+          @if (isSystemAdmin) {
+            <button mat-menu-item id="hangfire-dashboard-btn" [disabled]="!isAppOnline" (click)="hangfireDashboard()">
+              <mat-icon>pending_actions</mat-icon>
+              {{ t("hangfire_dashboard") }}
```

Comment:
I realize you're just following the pattern, but as I was about to upload the latest strings to Crowdin, I'm wondering how translators are supposed to translate this, or why we would ask them to. Same could be said for a couple of the others.